test/utils/Defaults.sol

```diff
@@ -376,63 +376,87 @@ contract Defaults is Merkle {
     }
 
     /*//////////////////////////////////////////////////////////////////////////
-                                        BATCH
+                                    BATCH-LOCKUP
     //////////////////////////////////////////////////////////////////////////*/
 
-    /// @dev Returns a default-size batch of `Batch.CreateWithDurationsLD` parameters.
-    function batchCreateWithDurationsLD() public view returns (Batch.CreateWithDurationsLD[] memory batch) {
-        batch = BatchBuilder.fillBatch(createWithDurationsLD(), BATCH_SIZE);
+    /// @dev Returns a default-size batch of `BatchLockup.CreateWithDurationsLD` parameters.
+    function batchLockupCreateWithDurationsLD()
```

Comment:
That this is about `Lockup` is implied by `LD`. We can just say `batchCreateWithDurationsLD`, and ditto for the other functions in here.